app/src/pages/instance/allUsersPage/allUsersHeader/allUsersExport/allUsersExport.tsx

```
@@ -0,0 +1,73 @@
+});
+
+export const AllUsersExport = ({ appliedFiltersCount }: AllUsersExportProps) => {
+  const { formatMessage } = useIntl();
+  const { trackEvent } = useTracking();
+  const dispatch = useDispatch();
+  const query = useSelector(querySelector) as QueryParams;
+
+  const handleClick = () => {
+    const filterParams = { ...query, offset: 0 };
+    const data = prepareQueryFilters(filterParams);
+    delete data.limit;
```

Biome (1.9.4)

[error] Avoid the delete operator which can impact performance.

Unsafe fix: Use an undefined assignment instead.


(lint/performance/noDelete)